content/post/2019-05-08.md

```diff
@@ -0,0 +1,211 @@
+
+#### Left Sidebar
+
+Contains several tabs that allow file browsing, check the running kernels, show the command palette and verify the open tabs. Moreover, the panel can be expanded with optional extensions.
+
+#### Main Work Area
+
+In this area, documents (notebooks, images, consoles, datasets) are organized in panels or tabs that can be subdivided or resized to ease the workflow.
+
+
+
+## Working with Files
+
+#### Create a new file
+
+If we click on the `+` icon in the tab menu we will open a new Launcher tab in the main work area where we can select the type of file we want to create. These will depend on the number of available kernels installed in our system as well as other extension that we have incorporated to JupyterLab. The new files main include Jupyter Notebooks, consoles, terminals, diagrams, Latex files, etc.
+
+![](C:\Users\user\Documents\GitHub\codingclubuc3m.netlify.com\static\post\2019-05-08_files\3.png)
+
+#### Opening or closing files 
+
+The `File` tab includes several options to create, save and export files depending on their nature.
+
+![](C:\Users\user\Documents\GitHub\codingclubuc3m.netlify.com\static\post\2019-05-08_files\4.png)
+
+## Text Editor
+
+JupyterLab includes a highly configurable text editor that includes syntax highlighting and automatic indentation, compatible with several programing languages.
+
+![](C:\Users\user\Documents\GitHub\codingclubuc3m.netlify.com\static\post\2019-05-08_files\5.png)
+
+## Jupyter Notebooks
+
+One of the main capabilities of JupyterLab is that it offers full support to work with  [Jupyter Notebooks](https://jupyter-notebook.readthedocs.io/en/stable/) (`.ipynb` files). In this type of files we can combine runnable code (R, Python, Matlab, C++, etc.) with text (Markdown), LaTeX equations, images and interactive graphs.
+
+![](C:\Users\user\Documents\GitHub\codingclubuc3m.netlify.com\static\post\2019-05-08_files\6.png)
+
+In this session, we will explore some of the main capabilities of Jupyter Notebooks and their potential use in research and teaching. On top of this, JupyterLab enables new functionalities to the notebooks that include:
+
+- Drop and drag cells within the notebook
+- Drag cells between notebooks
+- Expand and collapse cells
+- Create multiple views (synchronized) of a notebook
+- Connect a code console to a notebook kernel
+- Tab autocompletion in the editor
+- Change themes
+
+## Code Consoles
+
+This functionality allows running code interactively in a kernel. The cells also include rich output such as Tab-completion.
+
+![](C:\Users\user\Documents\GitHub\codingclubuc3m.netlify.com\static\post\2019-05-08_files\7.png)
+
+
+
+## Extensions
+
+JupyterLab is highly customizable through [extensions](https://jupyterlab.readthedocs.io/en/stable/user/extensions.html), that can provide new file viewers, editors, themes, etc.  The first step to install new extensions, is to install `Node.js`:
```

Comment:
Creo que es mejor sin la "coma" en la última frase:

... to install new extensions is to install Node.js